src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,225 @@
+package ui;
+
+import util.TaskList;
+import tasks.Task;
+
+import java.util.Scanner;
+import java.lang.StringBuilder;
+
+/**
+ *  Encapsulates a user interface handler that deals with interactions with
+ *  the user.
+ *
+ *   @author atharvjoshi
+ *   @version CS2103 AY19/20 Sem 1 iP Week 4
+ */
+public class Ui {
+    /** welcome message. */
+    private String welcomeMessage;
+
+    /** logo for Duke. */
+    private String logo;
+
+    /** exit message. */
+    private String exitMessage;
+
+    /**
+     *  Instantiates the user interface handler.
+     */
+    Ui() {
+        this.logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        this.welcomeMessage = "Hello! I'm Duke!!!\n";
+        this.exitMessage = "Hope to see you again soon!\n";
+    }
+
+    /**
+     * Returns the specified welcome message and logo.
+     *
+     * @return welcome message.
+     */
+    String getWelcomeMessage() {
+        return logo + welcomeMessage;
+    }
+
+    /**
+     * Returns the exit message that wishes the user goodbye upon exiting Duke.
+     *
+     * @return exit message.
+     */
+    public String getFarewellMessage() {
+        return exitMessage;
+    }
+
+    /**
+     * Returns startup error message.
+     *
+     * @return startup error message.
+     */
+    String showStartupError() {
+        return "Unable to load tasks from hard disk\n";
```

Comment:
inconsistent return - one is a variable and the other a string.
Consider using System.lineSeperator() instead to support all platforms instead of \n